### PR TITLE
Add: about missing types in RegExp

### PR DESCRIPTION
**Pull request by zusinShinpei**
Fixes #

### CODE REVIEW

**Comment by sandersn** [comment]
@zusinShinpei Can you file a bug that links to documentation for the missing types?

**Comment by zusinShinpei** [comment]
@sandersn https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/dotAll

File: lib/lib.es5.d.ts [hunk 1]
```diff
@@ -964,7 +973,11 @@ interface RegExpConstructor {
     $7: string;
     $8: string;
     $9: string;
+    input: string;
```
Comment (sandersn) [inline lib/lib.es5.d.ts:976]: These 4 properties are marked as non-standard. I don't want to make our list of non-standard extensions any bigger than it already is.

**Comment by sandersn** [comment]
This PR hasn't seen any activity for quite a while, so I'm going to close it to keep the number of open PRs manageable. I would recommend creating a bug so that we can decide what to do first.

**Pull request closed without merging**